npm/next/7.0/@n4jsd/next/lib/prefetch.n4jsd

```diff
@@ -1,5 +1,5 @@
 import * as React from "react";
-import { LinkProps } from "link";
+import { LinkProps } from "lib/link";
```

Comment:
lib? or next/lib/link? it misses the npm

Reply:
This is a local file, not a library.